products/views.py

```diff
@@ -1,3 +1,34 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http import JsonResponse
+from django.views import View
+
+from .models import Product, Image
+
+class ProductDetailView(View):
+    def get(self, request, **kwargs):
+        result = []
+        product = Product.objects.get(id=kwargs["product_id"])
+        colors = product.colors.all()
+        sizes = product.sizes.all()
+        images = Image.objects.filter(product=product.id)
```

Comment:
line 11~ line 14를 한줄로 줄여 주시고,  `images = Image.objects.filter(product=product.id)` 코드를 Product 부터 접근해서 가져오도록 수정해주세요~

```suggestion
        product = Product.objects.get(id=kwargs["product_id"])
        colors = product.colors.all()
        sizes = product.sizes.all()
        images = Image.objects.filter(product=product.id)
```

Reply:
수정했습니다!